### PR TITLE
chore: minor optimization for long task lists

### PR DESCRIPTION
**Pull request by dsalahutdinov**

### CODE REVIEW

File: list.go [hunk 1]
```diff
@@ -9,15 +9,17 @@ func List(c *cli.Context) error {
...
+	projectIds := make([]int, projectsCount, projectsCount)
+	for i, project := range client.Store.Projects {
+		projectIds[i] = project.GetID()
 	}
 	projectColorHash := GenerateColorHash(projectIds, colorList)
 	ex := Filter(c.String("filter"))
 
-	itemList := [][]string{}
-	for _, itemOrder := range client.Store.ItemOrders {
+  orderItemsCount := len(client.Store.ItemOrders)
```
Comment (sachaos) [inline list.go]: Please align indent

File: list.go [hunk 2]
```diff
@@ -9,15 +9,17 @@ func List(c *cli.Context) error {
...
+	itemList := make([][]string, orderItemsCount, orderItemsCount)
```
Comment (sachaos) [inline list.go]: `orderItemCount` is not enough.
It should filter out checked task and tasks which rejected by filter option.

File: list.go [hunk 3]
```diff
@@ -9,15 +9,17 @@ func List(c *cli.Context) error {
 	client := GetClient(c)
 
 	colorList := ColorList()
-	var projectIds []int
-	for _, project := range client.Store.Projects {
-		projectIds = append(projectIds, project.GetID())
+	projectsCount := len(client.Store.Projects)
+	projectIds := make([]int, projectsCount, projectsCount)
```
Comment (sachaos) [inline list.go]: :+1:

**Comment by dsalahutdinov** [comment]
@sachaos , thank you for review. I've fixed bug (just made new array with 0 lengh and totalSize capacity), could you please check out

**Comment by sachaos** [comment]
@dsalahutdinov Cool! Thank you for your PR.

**Pull request merged**